code/game/data_huds.dm

```diff
@@ -224,7 +224,7 @@
 	var/icon/I = icon(icon, icon_state, dir)
 	holder.pixel_y = I.Height() - world.icon_size
 	holder.icon_state = "hudno_id"
-	if(wear_id)
+	if(wear_id?.GetID())
```

Comment:
just use get_idcard(FALSE)

Reply:
also what does the ?. operator do

Reply:
?. is a safe navigation operator. If the first obj is null it stops there and returns null for it, otherwise it continues on in the chain; so it's equivalent to writing if(wear_id && wear_id.getid())

Also, using get_idcard() would be a major behavior change.